commands/push_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) OpenFaaS project 2018. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"testing"
+
+	"github.com/openfaas/faas-cli/stack"
+)
+
+func Test_PushValidation(t *testing.T) {
+	testCases := []struct {
+		name     string
+		scenario string
+		image    string
+		isValid  bool
+	}{
+		{scenario: "Valid image username", name: "cli", image: "alexellis/faas-cli", isValid: true},
```

Comment:
another one for isValid: false?

Reply:
Yes we should have more scenarios.. this is for @BurtonR to get a head-start on the tests